share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,628 @@
+## Wake Tutorial
+
+### Invoking wake
+
+1. Create a wake database: `wake --init .`
+
+2. Run command for wake: `wake -vx '<function_name> <parameter_if_any>'`
+
+* If you want to display a string you can that using the below function by writing the string to be displayed inisde "" and call the function abc . Ex: wake -x 'abc'
+```
```

Comment:
This example is not tutorial-y enough. Are you saying "Create a file called abc.wake and hten put this stuff in it", then call "wake -x abc" or? What is the relationship between the `wake -x 'abc'` and the text shown here?

Reply:
No. abc is a function name. You can add all the functions in any file with `.wake` extension and execute it by calling `wake -x '<function name>'`

Reply:
I was lost here... and then eventually,  I figured out some way to do it. 
What I did was:
1. go to federation/scripts
2. put def abc definition in <any name>.wake in the directory
3. do ./runWake -x 'abc' and this will set up WAKE_ENV and load-wake and then call abc function defined in <any name>.wake

It would be good for beginners that these steps are written step by step and not force them to figure things out.

Reply:
Filename is not important. Whenever you call a function it will look for that  function in the wake file. 
If in case of multiple `.wake` files, wake is intelligent enough to parse through all `.wake` files and get the function and execute it.